content/theme/characters/archetypes/outlaws/thief.md

```diff
@@ -2,4 +2,41 @@
 title: "Thief"
 date: 2019-12-20T23:40:03-08:00
 draft: false
----
\ No newline at end of file
+layout: archetype
+
+attributes:
+  awareness: 11
+  coordination: 11
+  knowledge: 8
+  physique: 8
+  presence: 7
+  reflex: 9
+
+skills:
+  awareness:
+    conceal: 4
+    sneak: 4
+    hide: 4
+    disguise: 3
+    forge: 2
```

Comment:
seems like a big missed opportunity to use `pick lock`, and `steal`, considering those are the two bread-and-butter skills of any thief.